src/bin/rustfmt.rs

```diff
@@ -63,8 +61,8 @@ enum Operation {
 struct CliOptions {
     skip_children: bool,
     verbose: bool,
-    write_mode: Option<WriteMode>,
-    file_lines: FileLines, // Default is all lines in all files.
+    write_mode: Option<String>,
+    file_lines: Option<String>,
```

Comment:
It seems bad that everything is a String now. Could you have setter methods for each config option do that setting as well as getting is strongly typed?